### PR TITLE
Make all links internal

### LINKED ISSUE #29: Prepare documentation for internal review
Multiple variable names for the same type of object were used in tutorials. To make it easier for the user to utilize examples from multiple tutorials, these variable names should be standardized. Ditto for configuration settings where applicable. There are also some formatting errors and/or areas of improvement.

To prepare for an internal review of content:
- [x] Standardize variable names throughout tutorials
- [x] Standardize configuration settings (e.g. image size, max frames collected, etc.) except for where unique settings are needed to teach the concept
- [x] Review API documentation and improve descriptions
- [x] Adjust code comments so that they fit inside the window without horizontal scrolling
- [x] Review and improve formatting of tutorials
- [x] Add internal links to other tutorials where applicable
- [x] Implementation of homepage tiles
- [x] Move example JSON files and fix references
- [ ] Internal links instead of URLs

### PR DESCRIPTION
**Pull request by melissawm**
Also improves accessibility by using more descriptive link titles. See, for example, https://www.smashingmagazine.com/2012/06/links-should-never-say-click-here/ or https://www.accessguide.io/guide/descriptive-link-text

Closes #29

### CODE REVIEW

File: docs/tutorials/chunked.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 This tutorial will provide an example of writing chunked data to a Zarr storage device.
 
-Zarr has additional capabilities relative to the basic storage devices, namely _chunking_, _compression_, and _multiscale storage_. To enable _chunking_, set the attributes in an instance of the `ChunkingProperties` class. You can learn more about the Zarr capabilities in `Acquire` [here](https://github.com/acquire-project/acquire-driver-zarr).
+Zarr has additional capabilities relative to the basic storage devices, namely _chunking_, _compression_, and _multiscale storage_. To enable _chunking_, set the attributes in an instance of the `ChunkingProperties` class. You can learn more about the Zarr capabilities in `Acquire` in https://github.com/acquire-project/acquire-driver-zarr.
```
Comment (aliddell) [inline docs/tutorials/chunked.md:5]:
```suggestion
Zarr has additional capabilities relative to the basic storage devices, namely _chunking_, _compression_, and _multiscale storage_. To enable _chunking_, set the attributes in an instance of the `ChunkingProperties` class. You can learn more about the Zarr capabilities in `Acquire` in [the Acquire Zarr documentation](https://github.com/acquire-project/acquire-driver-zarr/blob/main/README.md).
```

File: docs/tutorials/compressed.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 This tutorial will provide an example of writing compressed data to a Zarr file.
 
 `Acquire` supports streaming compressed data to the `ZarrBlosc1*` storage devices. Compression is done via [Blosc](https://www.blosc.org/pages/blosc-in-depth/).
-Supported codecs are _lz4_ and _zstd_, available with **ZarrBlosc1Lz4ByteShuffle** and **ZarrBlosc1ZstdByteShuffle** devices, respectively. For a comparison of these codecs, please refer to the [Blosc docs](https://www.blosc.org/). You can learn more about the Zarr capabilities in `Acquire` [here](https://github.com/acquire-project/acquire-driver-zarr).
+Supported codecs are _lz4_ and _zstd_, available with **ZarrBlosc1Lz4ByteShuffle** and **ZarrBlosc1ZstdByteShuffle** devices, respectively. For a comparison of these codecs, please refer to the [Blosc docs](https://www.blosc.org/). You can learn more about the Zarr capabilities in `Acquire` in https://github.com/acquire-project/acquire-driver-zarr.
```
Comment (aliddell) [inline docs/tutorials/compressed.md:6]:
```suggestion
Supported codecs are _lz4_ and _zstd_, available with **ZarrBlosc1Lz4ByteShuffle** and **ZarrBlosc1ZstdByteShuffle** devices, respectively. For a comparison of these codecs, please refer to the [Blosc docs](https://www.blosc.org/). You can learn more about the Zarr capabilities in `Acquire` in [the Acquire Zarr documentation](https://github.com/acquire-project/acquire-driver-zarr/blob/main/README.md).
```

File: docs/tutorials/livestream.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Livestream to napari
 
-The below script can be used to livestream data to the [napari viewer](https://napari.org/stable/). You may also utilize the `Acquire` napari plugin, which is provided in the package upon install. You can access the plugin in the napari plugins menu once `Acquire` is installed. You can review the [plugin code here](https://github.com/acquire-project/acquire-python/blob/main/python/acquire/__init__.py#L131). You may also stream using other packages such at `matplotlib`.
+The below script can be used to livestream data to the [napari viewer](https://napari.org/stable/). You may also utilize the `Acquire` napari plugin, which is provided in the package upon install. You can access the plugin in the napari plugins menu once `Acquire` is installed. You can review the [plugin code in the `acquire-imaging` repository](https://github.com/acquire-project/acquire-python/blob/d4a927079830dd9b61289e19e68dafdff4e8e643/python/acquire/__init__.py#L134). You may also stream using other packages such at `matplotlib`.
```
Comment (aliddell) [inline docs/tutorials/livestream.md:3]: Is there a reason we're not using main here?

File: docs/tutorials/multiscale.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
 
 This tutorial will provide an example of writing multiscale data to a Zarr file.
 
-Zarr has additional capabilities relative to Acquire's basic storage devices, namely _chunking_, _compression_, and _multiscale storage_. To enable _chunking_ and _multiscale storage_, set those attributes in instances of the `ChunkingProperties` and `StorageProperties` classes, respectively. You can learn more about the Zarr capabilities in `Acquire` [here](https://github.com/acquire-project/acquire-driver-zarr).
+Zarr has additional capabilities relative to Acquire's basic storage devices, namely _chunking_, _compression_, and _multiscale storage_. To enable _chunking_ and _multiscale storage_, set those attributes in instances of the `ChunkingProperties` and `StorageProperties` classes, respectively. You can learn more about the Zarr capabilities in `Acquire` in https://github.com/acquire-project/acquire-driver-zarr.
```
Comment (aliddell) [inline docs/tutorials/multiscale.md:5]:
```suggestion
Zarr has additional capabilities relative to Acquire's basic storage devices, namely _chunking_, _compression_, and _multiscale storage_. To enable _chunking_ and _multiscale storage_, set those attributes in instances of the `ChunkingProperties` and `StorageProperties` classes, respectively. You can learn more about the Zarr capabilities in `Acquire` in [the Acquire Zarr documentation](https://github.com/acquire-project/acquire-driver-zarr/blob/main/README.md).
```

File: docs/tutorials/setup.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 # Utilizing the Setup Method
 
-This tutorial will provide an example of utilizing the [setup method](https://github.com/acquire-project/acquire-python/blob/ce0764e86bf1d10eb3a447eb670c28808ab3da3b/python/acquire/__init__.py#L41) to configure `Runtime` and specify some basic properties.
+This tutorial will provide an example of utilizing the [setup method][acquire.setup] to configure `Runtime` and specify some basic properties.
```
Comment (aliddell) [inline docs/tutorials/setup.md:3]: Link appears to be missing here

File: docs/tutorials/storage.md [hunk 6]
```diff
@@ -34,7 +34,7 @@ The output of that script will be:
 
 `Acquire` supports streaming data to [bigtiff](http://bigtiff.org/) and [Zarr V2](https://zarr.readthedocs.io/en/stable/spec/v2.html).
 
-Zarr has additional capabilities relative to the basic storage devices, namely _chunking_, _compression_, and _multiscale storage_. You can learn more about the Zarr capabilities in `Acquire` [here](https://github.com/acquire-project/acquire-driver-zarr).
+Zarr has additional capabilities relative to the basic storage devices, namely _chunking_, _compression_, and _multiscale storage_. You can learn more about the Zarr capabilities in `Acquire` in https://github.com/acquire-project/acquire-driver-zarr.
```
Comment (aliddell) [inline docs/tutorials/storage.md:37]:
```suggestion
Zarr has additional capabilities relative to the basic storage devices, namely _chunking_, _compression_, and _multiscale storage_. You can learn more about the Zarr capabilities in `Acquire` in [the Acquire Zarr documentation](https://github.com/acquire-project/acquire-driver-zarr/blob/main/README.md).
```

**Review by aliddell — COMMENTED** [review]
Some suggestions you can take or leave, and a question.

File: docs/tutorials/livestream.md (hunk 3 above)
Comment (melissawm) [reply docs/tutorials/livestream.md:3]: I just used a permalink as I believe that is the best way to make sure we'll keep linking to the correct line even if things are moved, but I admit I'm not too sure if that is required.

File: docs/tutorials/setup.md (hunk 5 above)
Comment (melissawm) [reply docs/tutorials/setup.md:3]: This syntax will actually link to the generated anchor to the `acquire.setup` function in the API documentation. See https://mkdocstrings.github.io/usage/#cross-references

Comment (aliddell) [reply docs/tutorials/setup.md:3]: Oh neat.

File: docs/tutorials/livestream.md (hunk 3 above)
Comment (aliddell) [reply docs/tutorials/livestream.md:3]: On second thought this is better.

**Review by aliddell — APPROVED** [review]

**Pull request merged**